### PR TITLE
Support Article Migration

### PR DESCRIPTION
**Pull request by mnmlluke**
# Please Do Not Merge Until Support Verify Items.

## Support:

- Make Sure URLs/links are working correctly
- Make sure the formatting is correct
- Notify @mnmlluke if the docs need fixing or removed or merged together

## Docs Team:

- I have added Troubleshooting to the main page.
- I Modified the side navigation for Troubleshooting.
- I updated links to go to the correct location for articles
- Please very if there is anything else, as Support will utilize troubleshooting for KB Articles.

### CODE REVIEW

File: _docs/integrations/kubernetes.md [hunk 1]
```diff
@@ -583,7 +583,7 @@ kubectl create clusterrolebinding default-admin --clusterrole cluster-admin --se
 
 ### Kubernetes cluster - using an external reverse proxy (edge case)
 
-In case you're using an external reverse proxy to manage inbound traffic to your Kubernetes API, please read [this article]({{site.baseurl}}/docs/troubleshooting/common-issues/verify-cluster-tls-ssl-configuration/) to make sure your certificate setup are managed correctly in order to add your cluster successfully to Codefresh.
+In case you're using an external reverse proxy to manage inbound traffic to your Kubernetes API, please read [this article]({{site.baseurl}}/docs/troubleshooting/kb-articles/verify-cluster-tls-ssl-configuration/) to make sure your certificate setup are managed correctly in order to add your cluster successfully to Codefresh.
```
Comment (NimRegev) [inline _docs/integrations/kubernetes.md:586]: Suggest this rewrite to make it shorter:
If you are using an external reverse proxy to manage inbound traffic to your Kubernetes API, for correct certificate setup, see [this article]({{site.baseurl}}/docs/troubleshooting/kb-articles/verify-cluster-tls-ssl-configuration/).

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+group: troubleshooting
+sub-group: kb-articles
+toc: true
+kb: true
+common: false
+categories: [Pipelines]
+support-reviewed: 2023-04-18 LG
+---
+
+## Overview
```
Comment (NimRegev) [inline _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:13]: Change this to: Issue

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+An image using .Net Core is unexpectedly slow.
+
+## Details
+
+The official Docker images for .Net Core occasionally have performance issues.
+
+1. Attempt to run the image and tests locally.
+   1. If the issue persists, continue.
```
Comment (NimRegev) [inline _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:22]: 1 and 2 are not clear. 
1. What does continue mean - to continue with step 2
2. If it doesn't persist, why should we refer them to performance troubleshooting?

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+common: false
+categories: [Pipelines]
+support-reviewed: 2023-04-18 LG
+---
+
+## Overview
+
+An image using .Net Core is unexpectedly slow.
+
+## Details
```
Comment (NimRegev) [inline _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:17]: Would change this to Possible Causes

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+The official Docker images for .Net Core occasionally have performance issues.
+
```
Comment (NimRegev) [inline _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:20]: Add "Solution" title

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+1. Attempt to run the image and tests locally.
+   1. If the issue persists, continue.
+   2. If the issue does not persist, see our performance troubleshooting articles.
+2. Create a custom image including only the SDK and any other tools required.
+
+This allows a smaller image and is often known to create better performance.
+
+>_**Note** :_ This does not apply to all versions of Dotnet. 3.1 is known to occasionally have this problem.
```
Comment (NimRegev) [inline _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:28]: I would move the Note to Possible Causes

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md (hunk 4 above)
Comment (mnmlluke) [reply _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:17]: done

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md (hunk 5 above)
Comment (mnmlluke) [reply _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:20]: done

File: _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md (hunk 3 above)
Comment (mnmlluke) [reply _docs/troubleshooting/kb-articles/net-core-unexpectedly-slow.md:22]: Well i do not see the performance troubleshooting article SO removing this line.

Updated to say go to step 2

**Pull request merged**